frontend/src/@types/bot.d.ts

```diff
@@ -39,9 +39,18 @@ export type BotListItem = BotMeta & {
   available: boolean;
 };
 
+export type BotGenerationConfig = {
+  maxTokens: number;
+  topK: number;
+  topP: number;
+  temperature: number;
+  stopSequences: string[];
+}
+
 export type BotDetails = BotMeta & {
   instruction: string;
   embeddingParams: EmdeddingPrams;
+  generationConfig?: BotGenerationConfig;
```

Comment:
As well as EmbeddingParams, we don't want to care null for output (returned) values from api

Reply:
(See review comment for backend)